app/gui/src/dashboard/hooks/backendHooks.ts

```diff
@@ -149,6 +149,7 @@ const INVALIDATION_MAP: Partial<
   updateAsset: ['listDirectory', 'listAssetVersions'],
   closeProject: ['listDirectory', 'listAssetVersions'],
   updateDirectory: ['listDirectory'],
+  getProjectDetails: ['listDirectory'],
```

Comment:
is this correct? (why does a query need to invalidate another query?)

Reply:
@somebody1234 
![CleanShot 2024-10-14 at 11 43 53@2x](https://github.com/user-attachments/assets/0b7ad53b-f20a-4311-806a-ec315395f40f)
getProjectDetails is a mutation

Reply:
technically yes, but logically it is a query, it's only a mutation because i was being silly

Reply:
see https://github.com/enso-org/enso/pull/11268/commits/c8200f652920c4efa181c6c8e5a9e316ee7c1018

Reply:
well, specifically, either way it's not supposed to *change* the backend's state - so logically it shouldn't be invalidating anything

Reply:
Agree